docs/PacketFence_Clustering_Guide.asciidoc

```diff
@@ -518,6 +520,162 @@ In order to see the current maintenance state on a node:
 
   # /usr/local/pf/bin/cluster/maintenance
 
+'''
+
+Maintenance and Operation
+--------------------------
+
+Shutting down a PacketFence Active/Active cluster of tree nodes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to shutdown the cluster in a proper way, here is the steps you need to follow.
+We are here assuming that we are on a cluster of tree nodes, on Centos7.
+
+We will shutdown the nodes, one by one, starting by one of the "Slaves" and end with the "Master".
+
+Find the Master Server: ::
+* Type *ip a* on each nodes, and look for the one which hold the VIP (the /32 address).
```

Comment:
Instead, tell them to connect to the admin interface on the virtual IP, click on the '?' in the top right and it will show the server name there that is holding the VIP